packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -27,12 +28,13 @@ const {
  * can be considered equivalents (without identity discontinuity).
  */
 const ESGlobalKeys = new Set([
-    // *** 18.1 Value Properties of the Global Object
+    // *** 19.1 Value Properties of the Global Object
+    'globalThis',
```

Comment:
Should we export these https://github.com/salesforce/near-membrane/blob/master/packages/near-membrane-base/src/intrinsics.ts#L29 and use them here (I noticed `globalThis` is not part of the list there or maybe not the first and missed it there)